### PR TITLE
Add ACF field to onClick event

### PR DESCRIPTION
**Pull request by wilson1000-MoJ**

### CODE REVIEW

**Comment by BNewing** [comment]
Good spot! 

Just been taking a look at this: https://css-tricks.com/the-current-state-of-telephone-links/#article-header-id-4

It might be nice to include the country code. It looks like there are two ways of doing it - either prefix the number with +44, or switch the ACF field from number to text, so it'll accept/display the +. What do you think?

**Comment by wilson1000-MoJ** [comment]
That's a great idea. We can simply remove the 0 off the beginning and replace it with a +44 to keep things simple for the user. Sound ok?

**Comment by BNewing** [comment]
Sounds great :D

File: web/app/themes/brookhouse/header.php [hunk 1]
```diff
@@ -61,7 +61,7 @@
                         src="<?= get_template_directory_uri(); ?>/dist/img/brookhouse-logo.svg"
                         alt="<?php bloginfo('name'); ?>" class="site-header__logo"></a>
                 <div class="site-header__phone-number">
-                    <a href="tel:02076334149">
+                    <a href="tel:<?= get_field('telephone_number', 'option') ?>">
```
Comment (brown-a2) [inline web/app/themes/brookhouse/header.php]: As a team we'll have to decide about using short tags in PHP. I'd be interested to hear the benefits to using them over just writing $echo. I don't see how it's saving any time and echo seems more readable?

**Comment by wilson1000-MoJ** [comment]
Closing this PR as a better solution has been applied to another PR

**Pull request closed without merging**